src/plugins/displayLayout/components/EditMarquee.vue

```diff
@@ -21,17 +21,28 @@
  *****************************************************************************/
 
 <template>
-        <!-- Resize handles -->
-        <div class="c-frame-edit" :style="style">
-            <div class="c-frame-edit__handle c-frame-edit__handle--nw"
-                 @mousedown="startResize([1,1], [-1,-1], $event)"></div>
-            <div class="c-frame-edit__handle c-frame-edit__handle--ne"
-                 @mousedown="startResize([0,1], [1,-1], $event)"></div>
-            <div class="c-frame-edit__handle c-frame-edit__handle--sw"
-                 @mousedown="startResize([1,0], [-1,1], $event)"></div>
-            <div class="c-frame-edit__handle c-frame-edit__handle--se"
-                 @mousedown="startResize([0,0], [1,1], $event)"></div>
-        </div>
+<!-- Resize handles -->
+<div
+    class="c-frame-edit"
+    :style="marqueeStyle"
+>
+    <div
+        class="c-frame-edit__handle c-frame-edit__handle--nw"
+        @mousedown="startResize([1,1], [-1,-1], $event)"
+    />
+    <div
+        class="c-frame-edit__handle c-frame-edit__handle--ne"
+        @mousedown="startResize([0,1], [1,-1], $event)"
+    />
+    <div
+        class="c-frame-edit__handle c-frame-edit__handle--sw"
+        @mousedown="startResize([1,0], [-1,1], $event)"
+    />
+    <div
+        class="c-frame-edit__handle c-frame-edit__handle--se"
+        @mousedown="startResize([0,0], [1,1], $event)"
+    />
+</div>
```

Comment:
@davetsay Did lint complain about the way markup was written?

Reply:
Also, can the markup not be indented from <template>?!

Reply:
> Also, can the markup not be indented from `<template>`?!

It can, but we've chosen not to. `<template>` is non-functional, and appears in every Vue component, meaning all of the functional markup will be indented for no good reason. We lose four spaces for nothing.